Automattic/wp-calypso
# Untangling: apply new design to Settings -> Administration -> Reset your site

Fixes https://github.com/Automattic/dotcom-forge/issues/9861

## Proposed Changes

This PR applies the global design layout to the "Reset your site" flow:

<img width="822" alt="image" src="https://github.com/user-attachments/assets/aee78386-2d04-4d3a-980d-724169dbff1e">

<img width="829" alt="image" src="https://github.com/user-attachments/assets/e3f1dd0d-d978-4755-afb1-ac9a6754fdcb">


<img width="817" alt="image" src="https://github.com/user-attachments/assets/60b6ee0b-dc47-4748-819d-033c118e255a">


> [!NOTE]  
> Highlights information that users should take into account, even when skimming.

This also updates the layout in production, i.e. when the flag is not enabled yet. I **think** it's fine, because the flow is isolated so overall it's still consistent on its own. I don't think it is worth it to further complicate the code to retain the old layout...

## Why are these changes being made?


pbxlJb-6ye-p2

## Testing Instructions

1. Go to `/sites`
2. Click a site
3. Go to Settings -> Administration
4. Click `Reset your site` button
5. Verify that you can complete the flow.
6. Regression: go to `/settings/general/:slug`, click Reset your site, at the bottom, and verify you can still complete the flow.

## Pre-merge Checklist
